bundles/bundles.go

```diff
@@ -153,7 +154,8 @@ func NewBundleInstaller(bc BundleConfig) (BundleInstaller, error) {
 type OCIImageExtractor struct{}
 
 func (e OCIImageExtractor) Install(config *BundleConfig) error {
-	return utils.ExtractOCIImage(config.Target, config.RootPath, config.LocalFile)
+	platform := fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH)
```

Comment:
feels like `fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH)` could be a function that receives the runtime and returns the platform